package.json

```diff
@@ -39,6 +39,7 @@
     "@dpc-sdp/ripple-grid": "file:packages/Atoms/Grid",
     "@dpc-sdp/ripple-hero-banner": "file:packages/Organisms/HeroBanner",
     "@dpc-sdp/ripple-icon": "file:packages/Atoms/Icon",
+    "@dpc-sdp/ripple-latest-news-list": "file:packages/Organisms/LatestNewsList",
```

Comment:
This is not "latest news list" component, should be "featured news" component.
I will prefer to have a "ripple-news" package for hosting it.

Reply:
![screen shot 2018-11-07 at 11 57 26 am](https://user-images.githubusercontent.com/12739575/48103682-51c18e80-e284-11e8-9956-58ef923e6ed0.png)

Can we please get these details on the ticket before development?